### PR TITLE
Fixing Issue 25 https://github.com/auth0-samples/auth0-android-sample/issues/25

### PR DESCRIPTION
**Pull request by iannyanes**

### CODE REVIEW

**Comment by mpaktiti** [comment]
@iannyanes I edited the formatting a bit, the quotes did not look good:

image

cc @chenkie

**Comment by chenkie** [comment]
Is this one ready to go?

**Comment by chenkie** [comment]
@iannyanes ^

**Comment by iannyanes** [comment]
Sorry @chenkie It is good to go. @mpaktiti edited what was needed.

File: articles/native-platforms/android/_includes/_callback_urls.md [hunk 1]
```diff
@@ -4,4 +4,12 @@
 
 > If you wish to only use DB connections, skip this point.
 
-The callback URLs are meant to be used to receive the OAuth response when logging in with social connections or enterprise connections. To set a callback URL, navigate to the [settings](${manage_url}/#/applications/${account.clientId}/settings) for your client application and include the URL in the "Callback URLs" text box.
+The callback URLs are meant to be used to receive the OAuth response when logging in with social connections or enterprise connections. To set a callback URL, navigate to the [settings](${manage_url}/#/applications/${account.clientId}/settings) for your client application and include the URL in the **Callback URLs** text box.
+
+To form the callback url for the sample projects, add the following url to your allowed callback URLs: `${account.namespace}/${account.appName}/<PACKAGE_NAME>/callback`.
```
Comment (chenkie) [inline articles/native-platforms/android/_includes/_callback_urls.md:9]: "url" should be uppercase everywhere. url => URL

**Pull request closed without merging**